docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: '3.2'
```

Comment:
maybe we can use this in e2e tests ? and run it on the same docker network as the backend?

Reply:
By same docker network do you mean same container?

Reply:
let us drop this suggestion might be too complicated.

By docker-network I mean https://github.com/CSCfi/beacon-network/blob/dev/docker-compose-test.yml#L5-L11